SU2_CFD/src/iteration_structure.cpp

```diff
@@ -2303,9 +2303,9 @@ void CDiscAdjFluidIteration::InitializeAdjoint(CSolver *****solver, CGeometry **
   bool heat = config[iZone]->GetWeakly_Coupled_Heat();
   bool interface_boundary = (config[iZone]->GetnMarker_Fluid_Load() > 0);
 
-  /*--- Initialize the adjoint of the objective function (typically with 1.0) ---*/
+//  /*--- Initialize the adjoint of the objective function (typically with 1.0) ---*/
   
-  solver[iZone][iInst][MESH_0][ADJFLOW_SOL]->SetAdj_ObjFunc(geometry[iZone][iInst][MESH_0], config[iZone]);
+//  solver[iZone][iInst][MESH_0][ADJFLOW_SOL]->SetAdj_ObjFunc(geometry[iZone][iInst][MESH_0], config[iZone]);
```

Comment:
Is this no longer used?

Reply:
The functionality is covered on the driver level by e.g. `CDiscAdjSinglezoneDriver::SetAdj_ObjFunction()`. I think `CDiscAdjSolver::SetAdj_ObjFunc` is just used by `CDiscAdjFSIDriver`. I'll delete the commented lines in a distinct commit to avoid confusion about it.